overlord/devicestate/firstboot.go

```diff
@@ -76,6 +76,24 @@ func installSeedSnap(st *state.State, sn *snap.SeedSnap, flags snapstate.Flags,
 	return snapstate.InstallPath(st, &sideInfo, path, "", sn.Channel, flags)
 }
 
+func checkSeedBases(snaps []*snap.Info) error {
```

Comment:
it would be good if some of the checks that `snap debug validate-seed` does would share code with this (or vice versa). See https://github.com/mvo5/snappy/blob/master/image/image.go#L792 and https://github.com/snapcore/snapd/pull/7105